observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,56 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6"]
+== Network Observability Operator 1.6.0
+The following advisory is available for the Network Observability Operator 1.6.0:
+// note to self: update errata live ID
+* link:https://access.redhat.com/errata/RHSA-2024:0853[Network Observability Operator 1.6.0]
+
+[id="network-observability-operator-1.6.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-custom-metrics-1.6"]
+==== Custom metrics API
+You can create custom metrics out of the flowlogs data using the `FlowMetrics` API. Flowlogs data can be used with Prometheus labels to customize cluster information on your dashboards. For more information, see 
+
+[id="network-observability-eBPF-performance-enhancements-1.6"]
+==== eBPF performance enhancements
+*NetObserv / Health* dashboard integration with eBPF metrics allows you to view alerts if the Network Observability eBPF agent hashmap table is full, and the eBPF agent processes flows with degraded performance, or when the capacity limiter is triggered. 
```

Comment:
Is it an `or` before `the eBPF agent processes flows with degraded performance`?

Reply:
It isn't. Hrm. Its like there's two cases where you see this eBPF alert:
1. When the eBPF agent hashmap table is full, which causes the eBPF agent to process flows with degraded performance.
2. When the capacity limiter is triggered. 

I think I need to kinda tweak this. I can see how the conjunctions are a little confusing here.

Reply:
sounds good